ios/Codeforces Watcher/Application/features/users/cells/userAccount/UserAccountViewCell.swift

```diff
@@ -0,0 +1,137 @@
+import SwiftUI
+
+struct UserAccountViewCell: View {
+    
+    var avatar: String = ""
+    
+    var handle: String = ""
+    var name: String = ""
+    var rating: NSMutableAttributedString = "".attributed
+    var maxRating: NSMutableAttributedString = "".attributed
+    var rank: NSMutableAttributedString = "".attributed
+    var contribution: NSMutableAttributedString = "".attributed
+    var dateOfLastRatingUpdate: String = "".localized
+    
+    var onViewProfile: (String) -> Void = { _ in }
+    
+    var rankColor: UIColor {
+        getColorByUserRank(rank.string.lowercased())
+    }
+    
+    var body: some View {
+        VStack(alignment: .center, spacing: 6) {
+            VStack(alignment: .leading, spacing: 0) {
+                HStack(spacing: 0) {
+                    CircleImageViewNew(
+                        userAvatar: avatar,
+                        borderColor: getColorByUserRank(rank.string).swiftUIColor,
+                        size: (80, 80)
+                    )
+                    
+                    Spacer()
+                        .frame(width: 18)
+                    
+                    HStack(spacing: 6) {
```

Comment:
Hm, the common practice is having an icon and text in the same group, and then this group repeated the needed number of times. It's easier to align and extract as a component.